apps/web/src/features/tx-notes/index.tsx

```diff
@@ -1,7 +1,12 @@
 import { featureToggled, FEATURES } from '@/utils/featureToggled'
 import { TxNote as TxNoteComponent } from './TxNote'
 import { TxNoteForm as TxNoteFormComponent } from './TxNoteForm'
+import { MODALS_EVENTS, trackEvent } from '@/services/analytics'
 
 export const TxNote = featureToggled(TxNoteComponent, FEATURES.TX_NOTES)
 export const TxNoteForm = featureToggled(TxNoteFormComponent, FEATURES.TX_NOTES)
 export * from './encodeTxNote'
+
+export function trackAddNote() {
+  trackEvent(MODALS_EVENTS.ADD_TX_NOTE)
+}
```

Comment:
nit: Not sure if we need this abstraction. Everywhere else we use trackEvent directly.